irods/manager/user_manager.py

```diff
@@ -30,7 +30,7 @@ def create(self, user_name, user_type, user_zone="", auth_str=""):
         message_body = GeneralAdminRequest(
             "add",
             "user",
-            user_name,
+            user_name if not user_zone else "{}#{}".format(user_name,user_zone),
```

Comment:
can we flip it around?    would remove the 'not', arguably slightly easier to read...

```
"{}#{}".format(user_name, user_zone) if user_zone else user_name,
```

Reply:
This line was central to testing problems I had this morning... due the changes I had to make subsequently, I think this logic may end up scanning better.